src/main/java/it/gov/pagopa/pu/classification/service/AssessmentsService.java

```diff
@@ -14,14 +20,41 @@
 public class AssessmentsService {
 
   private final InstallmentNoPIIService installmentNoPIIService;
+  private final IngestionFlowFileService ingestionFlowFileService;
+  private final DebtPositionTypeOrgService debtPositionTypeOrgService;
+  private final AssessmentsRepository assessmentsRepository;
 
-
-  public AssessmentsService(InstallmentNoPIIService installmentNoPIIService) {
+  public AssessmentsService(InstallmentNoPIIService installmentNoPIIService, IngestionFlowFileService ingestionFlowFileService, DebtPositionTypeOrgService debtPositionTypeOrgService, AssessmentsRepository assessmentsRepository) {
     this.installmentNoPIIService = installmentNoPIIService;
+    this.ingestionFlowFileService = ingestionFlowFileService;
+    this.debtPositionTypeOrgService = debtPositionTypeOrgService;
+    this.assessmentsRepository = assessmentsRepository;
   }
 
   public List<InstallmentNoPIIResponse> getInstallmentsByReceiptId(Long receiptId, String accessToken) {
     return installmentNoPIIService.getByReceiptId(receiptId, accessToken);
   }
 
+  public Assessments getAssessment(InstallmentNoPIIResponse installmentNoPIIResponse, String accessToken) {
+    IngestionFlowFile ingestionFlowFile = ingestionFlowFileService.getIngestionFlowFile(installmentNoPIIResponse.getIngestionFlowFileId(), accessToken);
+    String debtPositionTypeOrgCode = debtPositionTypeOrgService.getDebtPositionTypeOrgByInstallmentId(installmentNoPIIResponse.getInstallmentId(), accessToken).getCode();
+
+
+    return Assessments.builder()
+      .organizationId(ingestionFlowFile.getOrganizationId())
+      .debtPositionTypeOrgCode(debtPositionTypeOrgCode)
+      .status(AssessmentStatus.NEW)
+      .assessmentName(ingestionFlowFile.getFileName() + "_" + debtPositionTypeOrgCode)
```

Comment:
Is it a requirement to have this form for this field? As explained, the ingestionFlowFile could not exist